documentation/modules/ostack-app-cred-auth.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="openstack-application-credential-authentication_{context}"]
+= Using application credential authentication with an {osp} source provider
+
+You can use application credential authentication instead of username/password authentication when you use {project-first} to migrate a virtual machine that has an {osp} source provider.
+
+{project-short} supports both of the following types of application credential authentication:
+
+* Application credential ID
+* Application credential name
+
+For each type of application credential authentication, you need to use data from OpenStack to create a `Secret` manifest.
+
+.Prerequisites
+
+Have an {osp} account.
+
+.Procedure
+
+. In the dashboard of the {osp} web console, click *Project > API Access*.
+. Expand *Download OpenStack RC file* and click *OpenStack RC file*.
+=
+The file that is downloaded, referred to here as `<openstack_rc_file>` includes the following fields used for application credential authentication:
++
+[source, terminal]
+----
+OS_AUTH_URL=http://192.168.100.181:5000
+OS_PROJECT_ID=dbed38bf449b495295868afa61a565dc
+OS_PROJECT_NAME="forklift"
+OS_DOMAIN_NAME="Default"
+OS_USERNAME="forklift"
+----
+
+. To get the data needed for application credential authentication, run the following command:
++
+[source,terminal]
+----
+$ openstack credential create --role member --role reader --secret redhat forklift
+----
++
+The output, referred to here as `<openstack_credential_output>`, includes:
++
+* The `id` (`applicationCredentialID` in the `Secret` manifest below) and `secret` (`applicationCredentialSecret` in the `Secret` manifest below) that you need for authentication using an application credential ID.
+* The `name` (`applicationCredentialName` in the `Secret` manifest below) and `secret` (`applicationCredentialSecret` in the `Secret` manifest below) that you need for authentication using an application credential name. You also must use `OS_DOMAIN_NAME` from `<openstack_rc_file>`if you use this method.
+
```

Comment:
@anarnold97 Style question: Do you think the parenthetical comments in lines 47-48 should be dropped? The documentation of the manifests below now contains the same information and I think  the comments, written in an earlier draft, now clutter the writing rather than adding useful information as they used to. WDYT?

Reply:
Remove. please

Reply:
@RichardHoch - changed